### PR TITLE
Upload fe #34

### PR DESCRIPTION
**Pull request by orjwan98**
for details: check commit history. 
references issue #34

### CODE REVIEW

**Review by amanshawar — APPROVED** [review]

**Review by Hadeel-Salamin — APPROVED** [review]
good job :+1:

File: client/src/components/Upload/index.js [hunk 1]
```diff
@@ -12,23 +17,39 @@ class Upload extends Component {
...
+      this.setState({ details: data });
+      this.props.history.push({ pathname: '/get-details', details: this.state.details });
+    });
   };
 
   render() {
     return (
       <React.Fragment>
         <Title />
         <Header />
```
Comment (jema28) [inline client/src/components/Upload/index.js]: The header text is missing. It should read 'Upload your photo'.

File: client/src/components/Upload/index.js [hunk 2]
```diff
@@ -12,23 +17,39 @@ class Upload extends Component {
...
-        <form>
-          <input label="upload file" type="file" onChange={this.handleUploadFile} />
-          <br />
-          <image id="logo" src={this.state.loc} alt="uploaded photo" height="300" width="300" />
-        </form>
+        <Instructions>
+          <TakeGoodPhoto>
```
Comment (jema28) [inline client/src/components/Upload/index.js:32]: Have a look at the style in Figma: the text is currently too large, it can go down a few pixels.

**Review by jema28 — CHANGES_REQUESTED** [review]
Compare with Figma and see comments: 
Screenshot 2019-03-26 at 11 38 01

File: client/src/components/Upload/index.js [hunk 3]
```diff
@@ -12,23 +17,39 @@ class Upload extends Component {
   handleUploadFile = (event) => {
     const img = new FormData();
     img.append('file', event.target.files[0]);
-    axios.post('/add-to-amazon', img)
-      .then(({ data }) => {
-        this.setState({ details: data });
-        this.props.history.push({ pathname: '/get-details', details: this.state.details });
-      });
+    axios.post('/add-to-amazon', img).then(({ data }) => {
```
Comment (jema28) [inline client/src/components/Upload/index.js:20]: Wasn't this already in master?

**Review by jema28 — CHANGES_REQUESTED** [review]

**Review by jema28 — APPROVED** [review]
👍

**Review by amanshawar — APPROVED** [review]

**Pull request merged**